RoddyCore/src/de/dkfz/roddy/core/Analysis.groovy

```diff
@@ -178,26 +181,26 @@ public class Analysis {
      * @param level The level for the context execution
      * @return
      */
-    public List<ExecutionContext> run(List<String> pidFilters, ExecutionContextLevel level, boolean preventLoggingOnQueryStatus) {
-        List<DataSet> selectedDatasets = getRuntimeService().loadDatasetsWithFilter(this, pidFilters);
-        List<ExecutionContext> runs = new LinkedList<>();
+    List<ExecutionContext> run(List<String> pidFilters, ExecutionContextLevel level, boolean preventLoggingOnQueryStatus) {
+        List<DataSet> selectedDatasets = getRuntimeService().loadDatasetsWithFilter(this, pidFilters)
+        List<ExecutionContext> runs = new LinkedList<>()
 
-        long creationCheckPoint = System.nanoTime();
+        long creationCheckPoint = System.nanoTime()
 
         for (DataSet ds : selectedDatasets) {
             if (level.allowedToSubmitJobs && !canStartJobs(ds)) {
-                logger.postAlwaysInfo("The " + Constants.PID + " " + ds.getId() + " is still running and will be skipped for the process.");
-                continue;
+                logger.postAlwaysInfo("The " + Constants.PID + " " + ds.getId() + " is still running and will be skipped for the process.")
```

Comment:
Could be changed to `Constants.DATASET`. We wanted to get away from the PID concept, right?